doc/teacher.md

```diff
@@ -163,22 +163,30 @@ This is the purpose of the `tests` repository.
 
 The file system layout of the `tests` repository must match that of the `assignments` repository, as shown below.
 The `assignment.yml` files contains the [assignment information](#assignment-information).
-In addition, each folder should also contain test code for the corresponding assignment.
+In addition, each assignment folder should also contain test code for the corresponding assignment.
+The `scripts` folder must contain a `run.sh` script file with commands to be executed when running assignment tests. This script will be used for every course assignment unless there is an assignment-specific `run.sh` script in the assignment folder. This script will only be used when running tests for this specific assignment.
```

Comment:
Please use a new line for each sentence.

I would rephrase it some: 
- The `scripts` folder may contain a `run.sh` script with commands ...
(must is too strong I think, since it is optional if all other assignments have a `run.sh` script).
- This script will only be used --> An assignment-specific `run.sh` script will only be used to run tests for the specific assignment.

Reply:
Updated.